src/pages/Lend/FilterAlpha/LendFilterAlpha.vue

```diff
@@ -178,12 +177,15 @@ export default {
 			return countryFilter;
 		},
 		resetFilter() {
+			this.gender = 'both';
+			this.sector = [];
+			this.country = [];
 			this.loanQueryFilters = {};
-			this.runQuery();
+			this.runQuery(this.loanQueryFilters);
 		},
-		runQuery() {
-			console.log('filters into fetchData:', this.loanQueryFilters);
-			fetchData(this.loanQueryFilters, this.apollo).then(flssData => {
+		runQuery(loanQueryFilters) {
+			console.log('filters into runQuery:', loanQueryFilters);
+			fetchData(loanQueryFilters, this.apollo).then(flssData => {
```

Comment:
Why the change here...was calling `fetchData(this.loanQueryFilters, ...` not working?

Reply:
it was working, and then I thought I should be explicit about which query filters were going in at which step and having it as a param would maybe let me pull out run query to the utils.  I'm happy to change it back. :)

Reply:
That makes sense. No change necessary